Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:13.8.0 as react-build
```

Comment:
This is a very common way to start - use an open source image that has most of what you need. However, if you're running lots of things it makes maintenance hard because you now essentially have a bunch of different operating systems running. For example, suppose nginx has a bug - if all your images start from the same base you patch the base image and rebuild the other images and you're done. But if you have a bunch of different base images you have to figure out how to patch each of them separately which can take a lot longer.

So we try to use the same base for all MVP studio stuff (sorry - I should have mentioned that). You can see our base images here: https://github.com/MVPStudio/docker-images. There's also some conventions discussed in that README. You probably want to use our node image: https://github.com/MVPStudio/docker-images/tree/master/node

Reply:
I chose mvpstudio/node:10 as the base image.